Assets/Scripts/Character/PlayerInput.cs

```diff
@@ -47,14 +48,14 @@ public class PlayerInput : BaseInput {
     if (dialogueManager && dialogueManager.isDialoging){
       return;
     }
-
     if (activeMovementInput) {
       movement.Move(Controls.Player.Move.ReadValue<Vector2>());
-      cameraBob.isBobbing = true;
+    }
+    if (isSprinting) {
+      viewBob.GLOBAL_BOB_SPEED_MULTIPLIER = 1.3f;
```

Comment:
This is a nitpick, but I think we should always try to avoid hard-coded values. So maybe instead of 1.3f we can have it as SPRINT_BOB_SPEED_MULTIPLIER which has the value of 1.3f